config.json

```diff
@@ -46,7 +46,7 @@
       "description": "MsBuild logging options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/flp:v=normal"
+      "defaultValue": "/flp:v=diag"
```

Comment:
I think the default value should be v=normal but the verbose option should be v=diag.